docs/content/zh/docs/concepts/components/core/builder-pod.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I think you are going to translate all these as we go further? Can we not add them to PR at all and only add translated content as we go - that way the PR is small and easy to manage please

Reply:
Yes going to translate all, and I got your idear to make it easier to review, but  `only add translated content as we go` makes extra work, because thus I need to copy one file at a time, make sure no wrong structure, and tanslate, and page wont appear the same because of lost(not copyed) pages, and more work when ref links to lost pages. so I suggest have a copy of both master and my PR repo, then delete zh folder in my pr repo, then use diff/merge tool to have all the other things verified; or rather, let's have it on another branch/repo, then we work on the none-master as the translation goes on, and only need to merge the fully translated and verify all